### DISCUSSION TITLE
FileOptimizer

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#29] [Requests]
This one might be worth a try: https://sourceforge.net/projects/nikkhokkho/

I found out about it from this website: https://imageoptim.com/versions.html

**Comment by mayrinck** [comment]
Well, guess there's no privacy policy for this one, so, I can't upload it...

**Reply by JuliaAgostino** [reply]
Is there an open-source license or anything like that?
